src/utils/fireQuery.ts

```diff
@@ -34,46 +26,6 @@ type FireQueryArgs = QueryArgs & {
 
 type FireQuery = (query: FireQueryArgs) => Promise<QueryResult[]>;
 
-type PredefinedSelection = {
```

Comment:
no notable changes here - just moved `predefinedSelections` to its own file for reuse